src/classes/User.ts

```diff
@@ -1,345 +1,247 @@
 /* eslint-disable no-unused-vars */
-import { PassageConfig } from "../types/PassageConfig";
-import axios from "../utils/axios";
-import { PassageError } from "./PassageError";
-
-interface UserEventInfo {
```

Comment:
Moved all of these into their own file in `/types` in line with other type defs

Reply:
Good call!